### PR TITLE
fix(curriculum): Learn CSS Variables by Building a City Skyline Step 40

### LINKED ISSUE #47568: Learn CSS Variables by Building a City Skyline Step 40 broken
### Describe the Issue

The test hints suggest applying a background, even though I am applying a background correctly.
After checking the MDN page here: https://developer.mozilla.org/en-US/docs/Web/CSS/background
I see that the 'background' property is shorthand for 'background-color'. As there is already a background-color property the new 'background' property must go after, however this isn't mentioned anywhere or given as a hint anywhere.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-variables-by-building-a-city-skyline/step-40

### Your code

This is invalid and gives the hint: "You should apply a background to .bb1a.":
```
.bb1a {
  width: 70%;
  height: 10%;
  background: linear-gradient(var(--building-color1), var(--window-color1));
  background-color: var(--building-color1);
}
```
This is valid and passes the tests:
```
.bb1a {
  width: 70%;
  height: 10%;
  background-color: var(--building-color1);
  background: linear-gradient(var(--building-color1), var(--window-color1));
}
```

### Expected behavior

I expect a hint about what I've done wrong.
If my code isn't passing because I'm writing the code in the wrong place then it would be helpful to know that.

### Screenshots

_No response_

### System

- Device: Mac book pro
- OS: MacOS Monterey 12.5.1
- Browser: Chrome Dev 
- Version: 107.0.5300.0 (Official Build) dev (arm64)

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by arantebw**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47568 

Changes made:
- Add a note about the *DRY* principle due to redundant properties (i.e., `background-color` and `background`) in a single rule.
- Update the first hint to indicate the recommended definition of the `background` property.
- Add an empty newline after the definition of the `background-color` property to imply the recommended location to define the `background` property.

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98f0.md [hunk 1]
```diff
@@ -135,6 +137,7 @@ body {
   width: 70%;
   height: 10%;
   background-color: var(--building-color1);
+  
 }
```
Comment (auchynnikau) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98f0.md:141]: We don't need an empty line there.

```suggestion
  background-color: var(--building-color1);
}
```

**Review by moT01 — COMMENTED** [review]
Thanks for the PR @arantebw 🎉 What do you think of my suggestions?

**Comment by arantebw** [comment]
@raisedadead if some of the checks are failing in the PR branch, merging the updates from the `main` branch is the way to resolve it, right?

**Comment by moT01** [comment]
Not usually @arantebw. It looks like something was added [here](https://github.com/freeCodeCamp/freeCodeCamp/pull/48900) to fix a broken test, so updating this branch brought in those changes so they could pass. Typically you need to look at why the test is failing. Sometimes it's just a random failure and re-running them will fix it, sometimes something is broken and needs fixing.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Review by raisedadead — APPROVED** [review]

**Pull request merged**